### DISCUSSION TITLE
Join 2 tables based on User Id foreign key

### DISCUSSION THREAD

**Discussion by NielsVeen** [supabase/supabase#15239] [Questions]
I have 2 tables blogs & user_meta both with a user_id column that is a foreign key reference for the auth.user.id table. I'm trying to join the blogs and user_meta table based on the user_id, but I can't find a way to do this. 

Anyone have any tips?

**Comment by GaryAustin1** [comment]
You really need to have a public profile (maybe your user_meta is it) table and have your other tables point to that id for fk's.  Then you can join with the API.  Auth schema can't be accessed by the API even for joins to flow thru.
